### ISSUE TITLE
Class 'MongoId' not found in php 7

### ISSUE THREAD

**Issue by denis-gorin** [as3io/short-mongo-id#1]
In php 7 version a new MongoDB extension is used.
So instead of legacy MongoId you should use MongoDB\BSON\ObjectID  as detailed on the http://php.net/manual/en/class.mongoid.php

line 5
`use \MongoDB\BSON\ObjectID  as MongoId; `

There is no alternative for `getInc` in MongoDB\BSON\ObjectID so I've tried to use `__toString` method

line 29
`$timestamp  = (Integer) ($identifier->getTimestamp() * 1000) + substr($identifier->__toString(), -3);`

**Comment by jasonlfunk** [comment]
Unfortunately, using the __toString() method will not produce unique enough results. :(

**Comment by zarathustra323** [comment]
Yes, unfortunately the `MongoDB\BSON\ObjectId` does not expose the same methods as the legacy driver. We'll need to investigate getting them added.
